testing/godot/depends

```diff
@@ -1,9 +1,20 @@
-scons make
-python make
-libXcursor 
-libXinerama 
-mesa 
-libX11 
-libXi 
+alsa-lib
```

Comment:
I have to plead ignorance here, I made the depends without taking into account that X and it's libs would be needed.